src/scss/_common.scss

```diff
@@ -4,10 +4,6 @@
   max-width: 215px;
   padding: 15px;
   text-align: left;
-  
-  > p {
-    white-space: pre-wrap;
-  }
```

Comment:
Is it ok to delete this? Won't be a problem when using markdown?

Reply:
Yes its ok, this style I added in before PR.